docs/source/install.ipynb

```diff
@@ -38,7 +38,7 @@
     "### Add-ons\n",
```

Comment:
Above there is a section that has `Woodwork allows users to install add-ons` + complete + Dask + Koalas. That should be moved under this section.

Reply:
fixed